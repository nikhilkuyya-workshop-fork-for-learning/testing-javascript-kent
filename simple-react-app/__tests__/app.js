import App from '../app'
import { render } from 'react-testing-library'
import React from 'react'

test('render', () => {
  const { container } = render(<App firstname="nikhil" lastname="kuyya" />)
  expect(container.firstChild).toMatchSnapshot()
})
