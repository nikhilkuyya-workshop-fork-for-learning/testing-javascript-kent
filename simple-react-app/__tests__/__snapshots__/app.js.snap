// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div>
  <input
    name="name"
    value="nikhil"
  />
  <p>
    nikhil
  </p>
</div>
`;
